modules/test/playwright/tests/object-web/objectActions.spec.ts

```diff
@@ -18,51 +18,77 @@ export const test = mergeTests(
 	objectPagesTest
 );
 
+interface CreatedEntities {
+	notificationTemplatesId: number[];
+	objectActionsId: number[];
+	objectDefinition: ObjectDefinition;
+}
+
+const createdEntities = {
+	notificationTemplatesId: [],
+	objectActionsId: [],
+	objectDefinition: {},
+} as CreatedEntities;
+
+test.beforeEach(async ({apiHelpers}) => {
+	const newObjectDefinition =
+		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
+			status: {code: 0},
+		});
+
+	createdEntities.objectDefinition = newObjectDefinition;
+});
+
+test.afterEach(async ({apiHelpers}) => {
+	await apiHelpers.objectAdmin.deleteObjectDefinition(
+		createdEntities.objectDefinition.id
+	);
+
+	for (const templateId of createdEntities.notificationTemplatesId) {
+		await apiHelpers.notification.deleteNotificationTemplate(templateId);
+
+		createdEntities.notificationTemplatesId = [];
+	}
+
+	for (const actionId of createdEntities.objectActionsId) {
+		await apiHelpers.objectAdmin.deleteObjectAction(actionId);
+
+		createdEntities.objectActionsId = [];
+	}
+});
+
 test.describe('manage object actions through object actions tab', () => {
```

Comment:
i know that you didn't change this line, but could you please change the description to use uppercase on the first word? like this:

`test.describe('Manage object actions through object actions tab'`

Reply:
sure, np! 🌷

Reply:
thanks! :cherry_blossom: